airbyte-ci/connectors/ci_credentials/README.md

```diff
@@ -68,31 +69,29 @@ pipx install --editable airbyte-ci/connectors/ci_credentials/
 
 This is useful when you are making changes to the package and want to test them in real-time.
 
-Note:
-
-- The package name is `pipelines`, not `airbyte-ci`. You will need this when uninstalling or reinstalling.
-- Even with the above `--editable` method, live changes to the code in the sibling project `/airbyte-ci/connectors/connector_ops/` are not automatically captured. To ensure you are using the latest code, use the command `pipx reinstall pipelines`.
+> [!Note]
+> - The package name is `ci_credentials`, not `airbyte-ci`. You will need this when uninstalling or reinstalling.
+> - Even with the above `--editable` method, live changes to the code in the sibling project `/airbyte-ci/connectors/connector_ops/` are not automatically captured. To ensure you are using the latest code, use the command `pipx reinstall ci_credentials`.
```

Comment:
`ci_credentials` does not depend on on `connector_ops`, why would we want to capture changes on `connector_ops`?

Reply:
I have little idea of what was about tbh. Let me cut that out then.